ush/hpssarch_gen.sh

```diff
@@ -584,5 +584,175 @@ if [ $type = "enkfgdas" -o $type = "enkfgfs" ]; then
+  echo  "${dirname}osense_${PDY}${cyc}_init.dat      " >>efsoi${CDUMP}.txt
+  echo  "${dirname}${head}abias_int.ensmean          " >>efsoi${CDUMP}.txt
+  if [ -s $ROTDIR/${dirpath}${head}cnvstat.ensmean ]; then
+       echo  "${dirname}${head}cnvstat.ensmean       " >>efsoi${CDUMP}.txt
+  fi
+  if [ -s $ROTDIR/${dirpath}${head}oznstat.ensmean ]; then
+       echo  "${dirname}${head}oznstat.ensmean       " >>efsoi${CDUMP}.txt
+  fi
+  if [ -s $ROTDIR/${dirpath}${head}radstat.ensmean ]; then
+       echo  "${dirname}${head}radstat.ensmean       " >>efsoi${CDUMP}.txt
+  fi
+  for FH in $nfhrs; do  # loop over analysis times in window
+     FHR=$(printf %03i $FH)
+     if [ $FHR -eq 6 ]; then
+        if [ -s $ROTDIR/${dirpath}${head}atmanl.ensmean${SUFFIX} ]; then
+            echo  "${dirname}${head}atmanl.ensmean${SUFFIX}      " >>efsoi${CDUMP}.txt
+	fi
+        if [ -s $ROTDIR/${dirpath}${head}atminc.ensmean${SUFFIX} ]; then
+            echo  "${dirname}${head}atminc.ensmean${SUFFIX}      " >>efsoi${CDUMP}.txt
+        fi
+     else
+        if [ -s $ROTDIR/${dirpath}${head}atma${FHR}.ensmean${SUFFIX} ]; then
+	    echo  "${dirname}${head}atma00${FHR}.ensmean${SUFFIX}      " >>efsoi${CDUMP}.txt
+        fi
+        if [ -s $ROTDIR/${dirpath}${head}atmi${FHR}.ensmean${SUFFIX} ]; then
+            echo  "${dirname}${head}atmi00${FHR}.ensmean${SUFFIX}      " >>efsoi${CDUMP}.txt
+        fi
+     fi 
+  done # loop over FHR
+  for fstep in ecenfsoi esfcfsoi eupdfsoi efcsfsoi eposfsoi ; do
+     echo  "logs/${CDATE}/${CDUMP}${fstep}*.log        " >>efsoi${CDUMP}.txt
+  done
+
+
+# Ensemble spread file only available with netcdf output
+  fh=6
+  while [ $fh -le 30 ]; do
+      fhr=$(printf %03i $fh)
+      echo  "${dirname}${head}atmf${fhr}.ensmean${SUFFIX}       " >>efsoi${CDUMP}.txt
+      echo  "${dirname}${head}sfcf${fhr}.ensmean${SUFFIX}       " >>efsoi${CDUMP}.txt
+      if [ $OUTPUT_FILE = "netcdf" ]; then
+          if [ -s $ROTDIR/${dirpath}${head}atmf${fhr}.ensspread${SUFFIX} ]; then
+	     echo  "${dirname}${head}atmf${fhr}.ensspread${SUFFIX}     " >>efsoi${CDUMP}.txt
+          fi
+      fi
+      fh=$((fh+6))
+  done
+
+  #...........................
+  n=1
+  while [ $n -le $NTARS ]; do
+     #...........................
+   
+     rm -f efsoi${CDUMP}_grp${n}.txt
+     rm -f efsoi${CDUMP}_restarta_grp${n}.txt
+     rm -f efsoi${CDUMP}_restartb_grp${n}.txt
+     touch efsoi${CDUMP}_grp${n}.txt
+     touch efsoi${CDUMP}_restarta_grp${n}.txt
+     touch efsoi${CDUMP}_restartb_grp${n}.txt
+   
+     m=1
+     while [ $m -le $NMEM_EARCGRP ]; do
+        nm=$(((n-1)*NMEM_EARCGRP+m))
+        mem=$(printf %03i $nm)
+        dirpath="efsoi${CDUMP}.${PDY}/${cyc}/atmos/mem${mem}/"
+        dirname="./${dirpath}"
+        head="${CDUMP}.t${cyc}z."
+    
+        #---
+        for FH in $nfhrs; do  # loop over analysis times in window
+           FHR=$(printf %03i $FH)
+           if [ $FHR -eq 6 ]; then
```

Comment:
Same

Reply:
Changing to reflect the scheme used by regular enkf in same file